parcel_test.go

```diff
@@ -28,60 +29,99 @@ func getTestParcel() Parcel {
 	}
 }
 
+func getDbConnection() *sql.DB {
+	db, err := sql.Open("sqlite", "tracker.db")
+	if err != nil {
+		fmt.Println(err)
+		return nil
+	}
+	return db
+}
+
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db := getDbConnection()
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
-
+	defer db.Close()
 	// add
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
-
+	id, err := store.Add(parcel)
```

Comment:
Можно сразу присвоить parcel.Number и убрать id

```suggestion
	parcel.Number, err := store.Add(parcel)
```

Reply:
Далее по коду аналогично

Reply:
переделал